functests/1_performance/performance.go

```diff
@@ -603,6 +603,10 @@ func verifyV1alpha1Conversion(v1alpha1Profile *performancev1alpha1.PerformancePr
 		return fmt.Errorf("spec RealTimeKernel field is different")
 	}
 
+	if (specRealTimeKernel.Enabled == nil) != (v1Profile.Spec.RealTimeKernel.Enabled == nil) {
```

Comment:
Thanks for finding it!
You may want to add the same check to the verifyV2Conversion, other than that lgtm

Reply:
Good catch, fixed.